client/src/components/atoms/DeletableTag.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FunctionComponent } from "react";
+import Tag from "./Tag";
```

Comment:
Unused import. Either add "Deletable" functionality to Tag.tsx or remove unused import.